hamster_gtk/overview/dialogs/export_dialog.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of 'hamster-gtk'.
+#
+# 'hamster-gtk' is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# 'hamster-gtk' is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with 'hamster-gtk'.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This Module provides the ``ExportDialog`` class to choose file to be exported."""
+
+from __future__ import absolute_import
+
+import os.path
+from enum import Enum
+from gettext import gettext as _
+
+from gi.repository import Gtk
+
+ExportType = Enum('ExportType', ('CSV', 'ICAL', 'XML'))
```

Comment:
Why do you use a 'Enum' here instead of just a dict?
Btw: Future iteration should be able to avoid this altogether as the backend will provide a registry for exactly this purpose. [LIB-232](https://projecthamster.atlassian.net/browse/LIB-232)